package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "possible-gazer-bookstore",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "start": "node ./bin/www"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/breyana/possible-gazer-bookstore.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/breyana/possible-gazer-bookstore/issues"
+  },
+  "homepage": "https://github.com/breyana/possible-gazer-bookstore#readme",
+  "dependencies": {
+    "bluebird": "^3.4.7",
+    "body-parser": "~1.15.2",
+    "cookie-parser": "~1.4.3",
+    "debug": "~2.2.0",
+    "express": "^4.14.1",
+    "mocha": "^3.2.0",
```

Comment:
I believe Mocha (and anything that is only used during the developing process) can go into your dev-dependencies, since it won't be used on your finished product side.